### PR TITLE
[BugFix] stop be without `-g` parameter

### PR DESCRIPTION
**Pull request by kevincai**
* kill be as quick as possible to restart the pod

## What type of PR is this:
- [X] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes:

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [X] I have checked the version labels which the pr will be auto backported to target branch
  - [X] 3.0
  - [X] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by imay** [comment]
@kevincai, in auto-scaling clusters, the addition and removal of BE happens frequently. To ensure successful execution of in-flight queries, a graceful exit is utilized.
So I think we should keep the graceful exit here.

**Comment by kevincai** [comment]
this pr is to remove graceful shutdown for be, not cn. cn graceful shutdown is still there with no change.

**Comment by kevincai** [comment]
and right now the be graceful shutdown doesn't work at all. need to kill -9 to force kill be process.

**Review by imay — APPROVED** [review]

**Comment by imay** [comment]
> and right now the be graceful shutdown doesn't work at all. need to kill -9 to force kill be process.

It should work even for Backend, @chaoyli can take a look.

**Comment by kevincai** [comment]
we can add this back when it works for be.

File: docker/dockerfiles/be/be_prestop.sh [hunk 1]
```diff
@@ -4,4 +4,4 @@ STARROCKS_ROOT=${STARROCKS_ROOT:-"/opt/starrocks"}
 STARROCKS_HOME=${STARROCKS_ROOT}/be
 
 # graceful stop be
-$STARROCKS_HOME/bin/stop_be.sh -g
+$STARROCKS_HOME/bin/stop_be.sh
```
Comment (chaoyli) [inline docker/dockerfiles/be/be_prestop.sh:7]: Why not call the `STARROCKS_HOME/bin/stop_be.sh` directly, instead of `be_prestop.sh`?

Comment (kevincai) [reply docker/dockerfiles/be/be_prestop.sh:7]: it is a wrapper script to be used in the k8s operator for the pod prestop hook. need to make the command as simple and stable as possible.

Comment (chaoyli) [reply docker/dockerfiles/be/be_prestop.sh:7]: If the stop_be.sh is not stable, the prestop.sh seems also unstable?

Comment (kevincai) [reply docker/dockerfiles/be/be_prestop.sh:7]: "stable" means the logic of the script. for example, if we want to have a graceful shutdown, we need to `stop_be.sh -g`, but we will make it in `pre_stop.sh` to call `stop_be.sh -g`, while to the operator, it still calls `prestop_be.sh` with no changes.

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request closed without merging**